platform/ios/display_server_ios.h

```diff
@@ -141,6 +141,9 @@ class DisplayServerIOS : public DisplayServer {
 	virtual void tts_resume() override;
 	virtual void tts_stop() override;
 
+	bool is_dark_mode_supported() const override;
```

Comment:
```suggestion
	virtual bool is_dark_mode_supported() const override;
```
We add both `virtual` and `override`, see the other methods here